frontend/docs/docs/deploy/customization.md

```diff
@@ -220,4 +220,6 @@ plausible.src = "https://plausible.io/js/script.js";
 plausible.defer = true;
 plausible.dataset.domain = "app.browsertrix.com";
 document.head.appendChild(plausible);
+
+window.analytics = window.plausible || function () { (window.plausible.q = window.plausible.q || []).push(arguments); };
```

Comment:
I don't think this is necessary: this only helps track 404 pages, and is an optional addition to Plausible's tracking script; and it also needs to follow the Plausible script itself, so it'd also need to be added to a new script tag and inserted after the main Plausible one.